templates/artist/artist_detail.html

```diff
@@ -200,6 +226,69 @@ <h1 class="header-font">{{ artist.name }}</h1>
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script type="text/javascript">
+
+    <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
+    <script type="text/javascript">
+
+
+
+        var doughnutData = [
+                {
+                    value: {{ campaign.fans_percentage }},
+                    color: "#46BFBD",
+                    highlight: "#5AD3D1",
+                    label: "Fan %"
+                },
+                {
+                    value: {{ campaign.artist_percentage }},
+                    color:"#F7464A",
+                    highlight: "#FF5A5E",
+                    label: "Artist %"
```

Comment:
I think it would be cool if this label used the actual artist's name. ie. `{{ artist.name|escapejs }} %`.